5_People/onboarding.md

```diff
@@ -34,6 +34,10 @@ Usually, it takes about ten working days to get through the onboarding issue, bu
 
 All Niteans issue invoices to the company. The invoice must include: your full name and address, company name and address, date of issue, invoice number, services done, amount and IBAN bank account. You can use an online [invoice generator](https://invoice-generator.com/) if you don't have your own.
 
+### Invoicing the company
+
+As a remote-first company, Niteans are from all around the globe. This means though that tax and accounting regulations differ per country. If you need help with setting your business, we might have Niteans who are from the same country, and can guide you on the necessary steps to undertake. 
```

Comment:
> All Niteans are hired as contractors and all responsibilities for taxes and accounting fall on you. This means you need to have a business or sole proprietorship open. If you need help with opening your business, we might have Niteans who are from the same country and can help you, so make sure to ask.

Reply:
From my slack question to @marbeann :

> There is no "paperwork" between an employee/contractor and Niteo, is there ?
> Sure we raise an invoice and niteo pays. But my “taxman” might ask if there is any “contract” between me and niteo (He hasn’t asked yet)
> So in case he asks, can Niteo provide one ? 

I think it needs at least the following things mentioned :
* Addresses of both parties
* Start date of the agreement
* <employee> will provide <services> and will raise an invoice.

I think some "one pager" boilerplate agreement could be enough.

Reply:
We can expand the IP agreement and add more info there. I wouldn't complicate it too much though. And it's not `employee` but `contractor`.